services/transformer/features.go

```diff
@@ -31,7 +34,8 @@ var defaultTransformerFeatures = `{
 	"routerTransform": {
 	  "MARKETO": true,
 	  "HS": true
-	}
+	},
+	"regulation": ["AM"],
```

Comment:
Why put this in default?

Reply:
Just thought of keeping something like previously, what do you suggest?